### PR TITLE
initial svg map

### PR DESCRIPTION
**Pull request by jordansread**
image

for #18

### CODE REVIEW

File: scripts/visualize/visualize_map.R [hunk 1]
```diff
@@ -0,0 +1,50 @@
+
+
+visualize.states_svg <- function(viz){
+  data <- readDepends(viz)
+  state.map <- data$`state-map`
```
Comment (jiwalker-usgs) [inline scripts/visualize/visualize_map.R:5]: didn't you convince me to use `[['']]`

File: viz.yaml [hunk 2]
```diff
@@ -27,6 +27,22 @@ info:
     tidyr:
       repo: CRAN
       version: 0.6.0
+    svglite:
```
Comment (jiwalker-usgs) [inline viz.yaml:30]: how are we feeling about this pattern?

File: scripts/visualize/visualize_map.R [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  state.name <- as.character(row.names(state.map)[state.map@plotOrder])
+  library(svglite)
```
Comment (jiwalker-usgs) [inline scripts/visualize/visualize_map.R:7]: `library` calls first and grouped together?

File: viz.yaml (hunk 2 above)
Comment (aappling-usgs) [reply viz.yaml:30]: fine with me so far

**Pull request merged**